docker/turing/Dockerfile

```diff
@@ -30,16 +18,15 @@ RUN apt-get update && apt-get install -y curl && \
   mkdir /data && \
   chown -R $USERNAME:$USERNAME /data && \
   ln -s /data /$USERNAME/.local/share/$BINARY && \
-  find /usr/bin/* /usr/sbin/* | grep -v curl | xargs rm
+  find /usr/bin/* /usr/sbin/* | grep -v 'curl\|sh\|chmod' | xargs rm
```

Comment:
This pattern includes a lot more than we'd want (`sh` matches many things).  I think our Docker image should be reviewed a bit more:

1. Smaller base image (not Alpine because of musl libc; maybe Debian), but maybe Debian's spent effort in making a minimal image since I've last used
2. Second image maybe? Ideally, we'd only need to run a `COPY` and a `RUN chmod` in CI.
3. Consider running our Debian pacakge.

I don't think we need to solve those here though..but we could.